__tests__/app.test.js

```diff
@@ -280,3 +280,25 @@ describe.only("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("DELETE /api/comments/:comment_id", () => {
```

Comment:
Try not to get into the habbit of pushing up code with any `.only`s on - if you're pushing code up in a work place everyone will want to know your code hasn't broken their code (so their tests still pass) before you push up!